library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileSplitter.java

```diff
@@ -53,6 +53,7 @@
    * This is a threshold on the no. of blocks emitted per window. A lot of blocks emitted
    * per window can overwhelm the downstream operators. This setting helps to control that.
    */
+  @NotNull
```

Comment:
@DT-Priyanka 
It is of type primitive int. So default value will be 0. Since the min is 1, it will be necessary for the user to set it. @NotNull is not relevant here.